docs/Style-Guide.md

```diff
@@ -0,0 +1,204 @@
+
+
+## Who is this guide for?
+
+This guide is for anyone who loves to build with Fastify or want to contribute to our documentation. You don't need to be an expert in writing technical documentation. Our guideline is here to help you.
+<br>
+
+Visit the [contribute](https://www.fastify.io/contribute) page on our website or read the [CONTRIBUTE.md](https://github.com/fastify/fastify/blob/master/CONTRIBUTING.md) file on Github to join our Open Source folks.
+
+
+## Before you write
+
+You need to know the following:
+
+* JavaScript
+* Node.js
+* Git
+* Github
+* Markdown
+* HTTP
+* NPM 
+
+
+### Consider your Audience
+
+
+Before you start writing, think about who your audience. In this case, your audience should already know HTTP, HTML, CSS, JavaScript, NPM and NodeJs. It's necessary to put your reader's mind because they are the one consuming your content. You want to give as much useful information as possible. Consider the vital things they need to know and how they can get them, make references and use words readers can relate with easily. As for feedback from the community, it can help you write better documentation that focuses on the user and what you want to achieve.
+
+
+### Get straight to the point
+
+
+Give your readers a clear and precise action to take. Start with what is most important for your readers. This way, you can help them find what they need faster. Mostly, readers tend to read the first content on a page, and many will not scroll further.
+
+**Example**
+
+Less like this: Colons are very important to register a parametric path. It lets the framework know a there is a new parameter created. You can place the colon before the parameter name for the parametric path can be created.
+
+More Like this: To register a parametric path, put a colon before the parameter name. Using colon lets the framework know it is a parametric path and not a wild card.
+
+
+
+<!-- <br> -->
+
+### Avoid adding video or image content 
+
+<!-- <br> -->
+
+Do not add videos or screenshots. Instead, make a referral link or a youtube video. You can add links by using `[Title](www.websitename.com)` in the markdown.
+
+**Example**
+
+```
+To learn more about hooks, See [Fastify hooks](https://www.fastify.io/docs/latest/Hooks).
+```
+
+Result:
+>To learn more about hooks, See [Fastify hooks](https://www.fastify.io/docs/latest/Hooks/).
+
+
+
+### Avoid plagiarism
+
+Make sure you evade copying other people's work, keep it as original as possible. You can learn from what they've done and also reference where it's from if you used a particular quote from their work.
+
+
+## Word Choice
+
+<!-- content here -->
+
+There are a few things you need to use and avoid when writing your documentation which will improve readability for readers and also make documentations neat, direct and clean all through your writing process.
+
+
+### Use the second person ("you")  as the pronoun
```

Comment:
This whole section does not read well, I think the title is off, maybe `When to use the second person "you" as the pronoun.

Reply:
Yes, that seems like a better title.